config/bundles.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+return [
```

Comment:
Is there any reasoning in the order of the bundles?

Reply:
same order we had before. If I remember we had also some issues if the order didn't match this case? /cc @wachterjohannes